<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnColor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFile">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSeason">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>season</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFont">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>font</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>87</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
